resources/lang/de/Config.properties

```diff
@@ -1,5 +1,6 @@
 ConfigurationTab=Plugin-Konfiguration
 isActiveLabel=Aktivieren
+termsAndConditions=Akzeptieren Sie die hier aufgeführten Allgemeinen Geschäftsbedingungen
```

Comment:
@kraath 
how can we place a link in this file that will lead to our terms and conditions page. ?

Reply:
Add link and test it.